extensions/positron-assistant/package.json

```diff
@@ -189,6 +189,36 @@
           "positron-assistant"
         ]
       },
+      {
+        "name": "inspectVariables",
+        "displayName": "Inspect Variables",
+        "modelDescription": "Inspect one or more variables in a console session, listing their child values, if any. Can be used to list the columns in a dataframe, the items in a column or array, or the elements of a list. Returns an array containing an array of children for each variable in the request. IMPORTANT: always use the access_key (even if it is a JSON string), NOT the display_name to refer to a variable. Example: for a session with identifier s-123 and variables [ { display_name: 'df', access_key: 'df-key' }, { 'display_name: 'foo', 'access_key': 'foo-key' } ], a request could be look like { 'sessionIdentifier': 's-123', 'accessKeys': [ ['df-key'], ['foo-key', 'bar-key'] ] }; the response would be an array of arrays, where each inner array contains the child variables of the corresponding variable in the request. The first inner array would contain the columns of the dataframe, and the second inner array would contain the elements of the list 'bar' inside the list 'foo'. Cannot be called when there is no session (the session identifier is empty or not defined). Can be called without any access keys to list all root-level variables in the session.",
```

Comment:
I wonder if there's a way to specify these over multiple lines 😅 atm I feel less likely to use long descriptions because of the single line limitation, even though I think it's probably very important to be descriptive.

Reply:
It is important to be descriptive but it also eats into the token budget a bit!